mondey_backend/src/mondey_backend/routers/milestones.py

```diff
@@ -52,25 +52,18 @@ def get_milestone_groups(
         current_active_user: CurrentActiveUserDep,
         child_id: int,
     ):
-        delta_months = 6
         child = get_db_child(session, current_active_user, child_id)
-
-        child_age_months = get_child_age_in_months(child)
+        milestone_answer_session = get_or_create_current_milestone_answer_session(
+            session, current_active_user, child
+        )
+        milestone_ids = list(milestone_answer_session.answers.keys())
+        print("milestone_ids", milestone_ids)
```

Comment:
is this useful for upcoming changes or is this debug output that slipped through the net?

Reply:
a print statement is always the latter, thanks :-)